client/src/components/Common/SearchProducts.js

```diff
@@ -140,7 +140,7 @@ export default function SearchProducts({ searchItem, searchInput }) {
     setSelectedCategory(e.target.value);
     axios
```

Comment:
move this axios.get into a function and reuse it everywhere to avoid duplicating code